tests/test_errors.py

```diff
@@ -199,28 +199,6 @@ def finalize(self, cfg: Any) -> None:
         ),
         id="dict:del_invalid_nested",
     ),
-    param(
-        Expected(
-            create=lambda: OmegaConf.create({"foo": "bar"}),
-            op=lambda cfg: cfg.__delattr__("not_found"),
-            key="not_found",
-            exception_type=ConfigKeyError,
-            msg="Key not found: 'not_found'",
-        ),
-        id="dict:delattr_invalid",
-    ),
-    param(
```

Comment:
Since `__delattr__` is no longer just an alias for `__delitem__`, does it make sense to put some of these tests back in?

Reply:
Yes - I think I've added back the tests that still apply. Now we allow deleting attributes from struct and structured,  some of the old tests (in the first commit) in `test_errors` does not apply to `__delattr__` any more since they are expecting the call to fail.

Reply:
The new behavior is not covered in `test_delattr` :)